performance/selfadj_perf.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from numpy import linalg
+from plotting import perf_plotting
+from timing import calc_timing
+
+if __name__ == "__main__":
+    # print(timeit.repeat(timer=t,repeat = 1000000, number = 1))
```

Comment:
Do we need to keep all the commented code in here?

Reply:
No, it can be removed.